codecov.yml

```diff
@@ -20,5 +20,6 @@ coverage:
     - "tuf/testutils/*"
     - "vendor/*"
     - "proto/*.pb.go"
+    - "trustmanager/remoteks/keystore.proto"
```

Comment:
Non-blocking - I think you may need to exclude the `*.pb.go` files, since the coverage will be calculated for the generated go files, not the proto files (I don't think codecov knows about protos)